### PR TITLE
this is still a heavy wip, but needed for testing because of how the mod works

### PR DESCRIPTION
**Pull request by DetroitWhiskey136**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [ ] Code changes have been tested against the Discord API and the discord.js wrapper, or there are no code changes
- [ ] Documentation has been added/modified, or there is nothing to change (docs/mods.json)

**Semantic versioning classification:**

- [ ] This PR changes DBM's interface (methods or parameters added to default methods)
- [ ] This PR **only** includes non-code changes, like changes to documentation, README, etc.

### CODE REVIEW

**Comment by DetroitWhiskey136** [comment]
I'll work on eslint errors while I wait

File: extensions/dbm_dashboard/dashboard_EXT.js [hunk 1]
```diff
@@ -295,55 +300,47 @@ module.exports = {
...
 
     Dashboard.storeData = function (fileName, dataName, data) {
       if (!fileName) return console.log('storeData("fileName", "dataName", "data")')
       if (!dataName) return console.log('storeData("fileName", "dataName", "data")')
 
       try {
-        if (!require('fs').existsSync(require('path').join(__dirname, 'dashboard_EXT'))) {
-          require('fs').mkdirSync(require('path').join(__dirname, 'dashboard_EXT'))
+        if (!fs.existsSync(require('path').join(__dirname, 'dashboard_EXT'))) {
+          fs.mkdirSync(require('path').join(__dirname, 'dashboard_EXT'))
```
Comment (almeidx) [inline extensions/dbm_dashboard/dashboard_EXT.js:331]:
```suggestion
        if (!fs.existsSync(path.join(__dirname, 'dashboard_EXT'))) {
          fs.mkdirSync(path.join(__dirname, 'dashboard_EXT'))
```

File: extensions/dbm_dashboard/dashboard_EXT.js [hunk 2]
```diff
@@ -352,16 +349,16 @@ module.exports = {
 
     Dashboard.retrieveData = function (fileName, dataName) {
       try {
-        if (!require('fs').existsSync(require('path').join(__dirname, 'dashboard_EXT'))) {
-          require('fs').mkdirSync(require('path').join(__dirname, 'dashboard_EXT'))
+        if (!fs.existsSync(require('path').join(__dirname, 'dashboard_EXT'))) {
+          fs.mkdirSync(require('path').join(__dirname, 'dashboard_EXT'))
```
Comment (almeidx) [inline extensions/dbm_dashboard/dashboard_EXT.js:353]:
```suggestion
        if (!fs.existsSync(path.join(__dirname, 'dashboard_EXT'))) {
          fs.mkdirSync(path.join(__dirname, 'dashboard_EXT'))
```

File: extensions/dbm_dashboard/dashboard_EXT.js [hunk 3]
```diff
@@ -370,54 +367,54 @@ module.exports = {
 
     Dashboard.retrieveFile = function (fileName) {
       try {
-        if (!require('fs').existsSync(require('path').join(__dirname, 'dashboard_EXT'))) {
-          require('fs').mkdirSync(require('path').join(__dirname, 'dashboard_EXT'))
+        if (!fs.existsSync(require('path').join(__dirname, 'dashboard_EXT'))) {
+          fs.mkdirSync(require('path').join(__dirname, 'dashboard_EXT'))
```
Comment (almeidx) [inline extensions/dbm_dashboard/dashboard_EXT.js:371]:
```suggestion
        if (!fs.existsSync(path.join(__dirname, 'dashboard_EXT'))) {
          fs.mkdirSync(path.join(__dirname, 'dashboard_EXT'))
```

**Review by almeidx — CHANGES_REQUESTED** [review]

**Pull request merged**